specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/share.json

```diff
@@ -0,0 +1,1796 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview Share Client",
+    "description": "Creates a data plane client for Purview Share.",
+    "version": "2022-06-30-preview"
+  },
+  "paths": {
+    "/receivedShares/{receivedShareId}": {
+      "get": {
+        "tags": [
+          "ReceivedShare"
+        ],
+        "summary": "Get a received share by unique id.",
+        "description": "Get a received share",
+        "operationId": "ReceivedShares_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/receivedShareId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ReceivedShare"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ReceivedShares_Get": {
+            "$ref": "./examples/ReceivedShares_Get.json"
+          },
+          "ReceivedShares_GetAdlsGen2Account": {
+            "$ref": "./examples/ReceivedShares_GetAdlsGen2Account.json"
+          },
+          "ReceivedShares_GetBlobAccount": {
+            "$ref": "./examples/ReceivedShares_GetBlobAccount.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ReceivedShare"
+        ],
+        "summary": "Put changes to a received share.",
+        "description": "Update changes to a received share",
+        "operationId": "ReceivedShares_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/receivedShareId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "receivedShare",
+            "description": "The updated received share",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReceivedShare"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
Long-running PUT should return 200 or 201, not 202.

Reply:
Done.